docs/features/software-catalog/user-guide.md

```diff
@@ -0,0 +1,99 @@
+- Tracking of real-time information sources
+- Dependency mapping
+- API exposure
+
+## Tracking Assets
+
+The recommended approach would be to represent information in catalog-info files, which the users themselves can manage. While automated classification based on repository contents can be helpful, it is recommended to use it only to generate the initial file and then allow humans to maintain it manually. The reason is that automation can sometimes fail, and it is essential to ensure the accuracy and reliability of this metadata. In short, humans should govern this piece of metadata to maintain its integrity.
+
+### Well-known trackable assets
+
+**Software Components**
+
+- Services
+- Websites
+- Libraries
+- Data Pipelines
+- Machine Learning Models
+- Third-party software components: It is recommended to have a separate repo for all 3d party catalog-info files.
+
+**Systems**
+
+- Jira installation
+- Pagerduty
+- Physical resources: This is probably more useful for longer-lived ones (For example servers).
+
+**Ownership - users - groups by**
+
+- Business units
+- Team
+- Product area
+
+**Naming strategies:**
+
+- Ldap: Internal ldap usernames as entity names. e.g., owner: user:myuser or user: my-team-name.
+
+**Ownership strategies:**
+
+**Ownership based on teams:** The concept of ownership in the system is team-centric; hence, the "owner" field must reference the LDAP name of a squad according to a prescribed set of regulations. While there may be instances where the ownership is attributed to an individual, such deviations might create challenges. In such cases, the user could receive a notification through the web interface, highlighting that this departure is an exception and needs rectification. To ensure adherence to this system, every entity in the model should have a designated owner, preferably a valid team within the LDAP hierarchy.
+
+**Ownership of features:** To track the ownership of specific features within your products and their interrelationships, two options are available: introducing a new component type, such as "feature," or creating a new Kind altogether. Nonetheless, it is advisable to choose the former approach and introduce a new type, like "feature," is advisable as it is less complicated and poses a lower risk.
+
+**LDAP doesn’t reflect the organization structure:** If Workday or a similar system serves as the source of truth, and LDAP is utilized for user attributes, it is advisable to develop a layer on top of various systems to establish a unified API that the entire organization can query, rather than only Backstage.
+
+**APIs**
+
+- OpenApi
+- AsyncApi
+- graphQL
+- gRPC
+
+APIs Could list networked services or libraries and it’s particularly useful to identify APIs that form boundaries between systems. There are some considerations to keep in mind:
+
+**API versions:** It’s not recommended to have multiple entity instances (e.g., metadata.name: my-api-v1 and metadata.name: my-api-v2).
```

Comment:
We actually do mention that explicitly when there are different major versions, those could be seen as different APIs and have dedicated entities for each.

But not for minor/patch level variations.

Reply:
Yes! Clear and concise.